src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1104,10 +1090,24 @@ Location GetDiagnosticLocation()
                     else if (MemberSignatureComparer.RuntimeImplicitImplementationComparer.Equals(interfaceMember, member) && !member.IsAccessor())
                     {
                         // CONSIDER: Dev10 does not seem to report this for indexers or their accessors.
-                        diagnostics.Add(ErrorCode.WRN_MultipleRuntimeImplementationMatches, member.Locations[0], member, interfaceMember, implementingType);
+                        diagnostics.Add(ErrorCode.WRN_MultipleRuntimeImplementationMatches, GetDiagnosticLocation(member), member, interfaceMember, implementingType);
                     }
                 }
             }
+
+            Location GetDiagnosticLocation(Symbol member)
```

Comment:
Please mark as private

Reply:
It's only a local function. I believe it's not possible to mark it with access modifiers.

Reply:
I'll put it at the beginning of the method to avoid confusion.